discovery-provider/src/tasks/tracks.py

```diff
@@ -117,6 +117,26 @@ def invalidate_old_track(session, track_id):
     ), "Update operation requires a current track to be invalidated"
 
 
+def update_credit_splits(session, track_record, track_metadata):
+    child_track_id = track_record.track_id
+
+    # Delete existing credits
+    session.query(Credit).filter_by(child_track_id=child_track_id).delete()
```

Comment:
I want to think through the delete and create strategy to make sure it's the best approach. Our notifications (actually a lot of our queries) use blocknumber checks as a proxy for time. This doesn't maintain that relationship, so if we wanted to do notif for a remix we'd need to join the tracks table with this table. 

Another option is to add two additional rows to this table, is_current and blocknumber. When we get a credit split, update all existing rows for that track to is_current=False, and write the new splits with is_current=True and blocknumber. That way we can query with blocknumber without a join. And also it becomes append only with a boolean "deleted" flag

Reply:
see other comment. resolved offline